src/validate_pyproject/plugins/__init__.py

```diff
@@ -102,7 +102,8 @@ def iterate_entry_points(group: str = ENTRYPOINT_GROUP) -> Iterable[EntryPoint]:
         # The select method was introduced in importlib_metadata 3.9 (and Python 3.10)
         # and the previous dict interface was declared deprecated
         select = typing.cast(
-            Any, getattr(entries, "select")  # noqa: B009
+            Any,
+            getattr(entries, "select"),  # noqa: B009
```

Comment:
I don't remember why I did not use `entries.select` directly here and instead used `getattr`, but I think we should be able to use it now...

```suggestion
            entries.select,
```

Reply:
I would expect it makes mypy happy, as we are having to work around supporting old versions here.

Reply:
I see, it makes sense. Thank you!